### PR TITLE
FIX #56 Create bundle from directory

### LINKED ISSUE #56: Create bundle from directory
As today it's possible to create bundles from archives (*.rpm, *.deb, *.tar.gz, *.zip), it would be great to use a directory as bundle source. Example:
 {
        "name": "my-bundle-from-directory",
        "version": "1.0",
        "files": [
          {
            "name": "my-bundle",
            "source": "overlay/etc",
            "destination": "/etc"
          }
        ]
      }

As result, all the content of overlay/etc is copied to /etc in the build image.

### PR DESCRIPTION
**Pull request by hugo6**
A tar.gz archive file is made from the directory

### CODE REVIEW

**Comment by lqueiroga** [comment]
It looks good for me ! But please add (or modify) a test

File: tests/integration/data/templateForBundleDirectoryTest.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "installation": {
+      "diskSize": 12288
+    },
+    "bundles": [
+      {
+        "name": "myBundle",
+        "version": "6.0",
+        "destination": "/tmp/myBundle",
+        "files": [
```
Comment (jbremond) [inline tests/integration/data/templateForBundleDirectoryTest.json]: We are using files section to handle a directory. About naming point of view it is a little bit strange. I would like to have @jgweir point of view.

**Comment by jbremond** [comment]
We need to update the hammr documentation inside this PR, in order to document this new feature

File: tests/integration/data/templateForBundleDirectoryTest.json (hunk 1 above)
Comment (jbremond) [inline tests/integration/data/templateForBundleDirectoryTest.json]: I have discuss with @jgweir about this and we are ok with the proposition.

**Comment by lqueiroga** [comment]
We are ok with code.
Wait @emuus review on documentation.

**Comment by emuus** [comment]
Ok for me. No typos spotted.

**Pull request merged**